app/views/applications/_show.html.haml

```diff
@@ -11,7 +11,11 @@
       Email
       %small (required)
     %td= @user.email
-
+  %tr
+    %td.left
+      Pronouns
+      %small (optional)
+      %p.small *This does not affect your application, Double Union wants to respect what pronouns you use
```

Comment:
Copy suggestion: "This does not affect your application. Double Union wants to respect the pronouns you use."